packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -229,6 +228,100 @@ describe('HttpMocker', () => {
             expect(response.body).toEqual((selectedExample as INodeExample).value);
           });
         });
+
+        describe('and the resource has an example, but the request does not specify any', () => {
+          it('should return the first viable example', () => {
```

Comment:
could we find a way to replace viable with some words which explain what that means? What constitutes viable?

Reply:
Actually it's just the first example.

Reply:
```suggestion
          it('should return the first example', () => {
```